leaf-ui/js/onFunctions.js

```diff
@@ -811,19 +850,13 @@ paper.on("link:options", function (cell) {
         for (let element of graph.getElements()) {
             var cell = element.findView(paper).model;
             var intention = cell.get('intention');
-            
-            if (intention != null) {
-                var initSatVal = intention.getUserEvaluationBBM(0).get('assignedEvidencePair');
-                if (intention.get('evolvingFunction') != null) {
-                    var funcType = intention.get('evolvingFunction').get('type');
-                }
-            }
```

Comment:
This is to prevent errors like this happening, tho idk if we want to directly limit the user's ability to do this 
![image](https://user-images.githubusercontent.com/47505178/164477503-8d07eddf-84ba-4891-967f-807405b4e7c4.png)

Reply:
Wait so are we not preventing this type of error anymore?

Reply:
Not sure if you guys put it back in as it still shows up here

Reply:
Should show up now!